### DISCUSSION TITLE
invite user not getting logged in keycloack. facing user_not_found issue  for the this api - /api/v1/Teacher/invite

### DISCUSSION THREAD

**Discussion by nishantfaye** [Sunbird-RC/community#273] [Issues]
this image show the invite user are created successfully  
![image](https://user-images.githubusercontent.com/30252514/192780703-f5e9cb47-8055-47b9-9ba6-85406258030f.png)
 but when try by url {keycloack-auth} /auth/realms/sunbird-rc/protocol/openid-connect/auth?scope=openid&response_type=code&redirect_uri=*&client_id=registry-frontend 
shows user not found error 
![image](https://user-images.githubusercontent.com/30252514/192781462-5d9494e9-e66b-417a-8cab-cb30199c59cb.png)
![image](https://user-images.githubusercontent.com/30252514/192780932-e331e42a-6988-4b7a-a879-f3c7a54618d9.png)

**Comment by tejash-jl** [comment]
@nishantfaye,
I would need more info to resolve this issue,
- can you check if the user is created in keycloak,
    - If Yes, have you set a password for the user.
    - If No, can you share the logs when you trigger the invite api.
- We have added support for only otp-based logins, you can change the login theme for `registry-frontend` client to `sunbird-rc` theme. By default it runs on mock otp layer, default otp is `1234`

**Reply by nishantfaye** [reply]
@tejash-jl  can you please help use to install sunbird-rc core because we are in initial stage of setup and facing issue with installation even to not able to get keycloak  admin console by following you given installation instruction from https://docs.sunbirdrc.dev/developer-documentation/installation-guide  and  by npm install --global registry-cli and check all docker images are running properly but facing such issue . for

**Reply by tejash-jl** [reply]
You can join the[ discord channel ](https://discord.gg/Q5mvw2mGC8) and ping me. We can join a call and resolve this
